README.md

```diff
@@ -1,7 +1,7 @@
 # zend-paginator
 
 [![Build Status](https://secure.travis-ci.org/zendframework/zend-paginator.svg?branch=master)](https://secure.travis-ci.org/zendframework/zend-paginator)
-[![Coverage Status](https://coveralls.io/repos/zendframework/zend-paginator/badge.svg?branch=master)](https://coveralls.io/r/zendframework/zend-paginator?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/zendframework/zend-paginator/badge.svg?branch=master)](https://coveralls.io/github/zendframework/zend-paginator?branch=master)
```

Comment:
Should the [template for the readme](https://github.com/zendframework/maintainers/blob/master/template/README.md) file be applied?

Reply:
Good catch! Thanks. Done now.